pages/index.js

```diff
@@ -1,7 +1,33 @@
+import useSWR from "swr";
+import Card from "@/components/Card";
+import styled from "styled-components";
+
 export default function HomePage() {
+  const { data } = useSWR("/api/plants");
+
+  if (!data) {
+    return <p>Loading...</p>;
+  }
+
   return (
     <div>
```

Comment:
this div prevents your Main grid from having any effect. I would suggest to remove it since it does nothing here.